### PR TITLE
docs: update README-pt-BR.md

### PR DESCRIPTION
**Pull request by gabrielcury**
docs: update README-pt-BR.md

### CODE REVIEW

File: .github/README-pt-BR.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Código e documentação copyright 2018 [B.C.Rikko](https://github.com/BcRikko).
 
 ## Desenvolvimento
 
-Se você quiser nos ajudar com o projeto, nós agradecemos contribuições de todos os tipos! Verifique nosso [`CONTRIBUTING-pt-BR.md`][contributing-document] para mais detalhes em como você pode ajudar a tornar o NES.css incrível!
+Se você quiser nos ajudar com o projeto, nós agradecemos as contribuições de todos os tipos! Verifique nosso [`CONTRIBUTING-pt-BR.md`][contributing-document] para mais detalhes como você pode ajudar a tornar o NES.css incrível!
```
Comment (omurilo) [inline .github/README-pt-BR.md:93]: Hey, dude, what's up?

I think when you say "para mais detalhes como você pode ajudar..." it is a confusing phrase, but if you say "para mais detalhes de como você pode ajudar...", it sounds better.

Comment (guastallaigor) [reply .github/README-pt-BR.md:93]: Hello. Yes, I agree.
But this PR is kind of dead for a year now, so if you want I think you can make a new one correcting all this mistakes (and other ones if you find those).
Thanks 👍

**Comment by guastallaigor** [comment]
Closing this because it's been open for a long time. But feel free to reopen if you want.

**Comment by Makegoodies1** [comment]
You on youtube with the game genie codes i been fixing messed up codes
trying to toss better ones so easy gameplay found a few lost MMO player
cause game not help player get stuff sent to me Developers are unkind
lately.

Michael Russell

**Pull request closed without merging**